tensorflow_addons/losses/__init__.py

```diff
@@ -22,10 +22,10 @@
 from tensorflow_addons.losses.giou_loss import giou_loss, GIoULoss
 from tensorflow_addons.losses.lifted import lifted_struct_loss, LiftedStructLoss
 from tensorflow_addons.losses.sparsemax_loss import sparsemax_loss, SparsemaxLoss
+from tensorflow_addons.losses.triplet import triplet_semihard_loss, TripletSemiHardLoss
```

Comment:
Can you put back those imports to where they were, it's better to keep the diff as small as possible.